guides/V1_AppActionCallback.md

```diff
@@ -0,0 +1,39 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'client-action-handler-callback'
+is: 'experimental'
+---
+
+# Client Action Handler Callback API
+Client action handlers allow your App to deliver a user experience to a Watson Work Services client. You will need to 
+register a call back URL that the client can invoke for specified actions so that the client can recognize and enable 
+your action for users. This callback API needs to be implemented for your app to process the client action. Once a user 
+invokes your action through the client, your App can access context. Learn more by reading 
+[Actions and Events](../guides/V1_ActionsEvents.md).
+
+## URI scheme
+_Host_ : **api.watsonwork.ibm.com**
+_Scheme_: **HTTPS**
+
+## Path
+```
+actionURL?actionHandlerContextToken=<token>
+```
+
+The **actionURL** is the one that was registered for the app. It is required to be a fully qualified URL using the 
+**https** protocol.
+
+The **actionHandlerContextToken** url parameter contains the client action handler context token which has to be used 
+to get the action context information via Watson Work Services, see 
+[Get action context information](./V1_GetActionContext.md) and also 
+[Actions and Events](../guides/V1_ActionsEvents.md). This action handler token is generated by Watson Work 
```

Comment:
Perhaps small clarification:

How about we change from:

>This action handler token is generated by Watson Work Services when a Watson Work Services client attempts to invoke a registered client action handler that has been registered by your app.

to:

This action handler token is generated by Watson Work Services as a result of a Watson Work Services client detecting that an app exists which could handle a particular user action. The client includes this token when it attempts to invoke the app's registered client action handler.

The other option is to eliminate the sentence as we are already pointing them to the general explanation doc.